pulpcore/cli/rpm/context.py

```diff
@@ -102,6 +102,17 @@ class PulpRpmRemoteContext(PulpRemoteContext):
         "sles_auth_token",
     }
 
+class PulpUlnRemoteContext(PulpRemoteContext):
```

Comment:
At some place i expect a `needs_plugin(PluginRequirement)` on the pulp_rpm version to introduce the uln remote. Probably in the `__init__` of this context.

Reply:
I added the requirement to remote function so that it only uses Uln when its above `3.12.0` version. https://github.com/ATIX-AG/pulp-cli/blob/0f2df1610202da5fe8274d10c1bacb901797a7cd/pulpcore/cli/rpm/remote.py#L39